### DISCUSSION TITLE
Retrospect v5.6.6

### DISCUSSION THREAD

**Discussion by basrieter** [retrospect-addon/plugin.video.retrospect#1722] [Announcements]
#### _NOTE: Python 2 is no longer supported by Retrospect. All Kodi users should use the Kodi Matrix or newer._

Added some SVT improvements.

### Framework related
_None_ 

### GUI/Settings/Language related
_None_

### Channel related
* Fixed: SVT Descriptions for some items (Fixes #1719).

<hr /><em>This discussion was created from the release <a href='https://github.com/retrospect-addon/plugin.video.retrospect/releases/tag/v5.6.6'>Retrospect v5.6.6</a>.</em>

**Comment by daspunner** [comment]
Hi!

Some TV-shows on TV4-Play will not show in Retrospect tough they do that in if I log in to www.tv4play.se. They are not premium-content and I have used the search function and looked for in looked manually in "TV Shows". I have seen this in the past with some content but now it is the show "Förrädarna". In the past there have been certain episodes of a show that where missing but this time it is the whole show. Any solution?

Here  is the show that I am missing:

https://www.tv4play.se/program/a8fab3e0543460e27180/f%C3%B6rr%C3%A4darna

/David

**Comment by basrieter** [comment]
See https://github.com/retrospect-addon/plugin.video.retrospect/issues/1714

**Comment by Marcmk1** [comment]
it looks like Dutch replay Chanels do not work anymore.
Is that correct?

**Reply by basrieter** [reply]
Yes, see #1730. A fix is almost there.
